lib/reactive_table.js

```diff
@@ -494,6 +497,12 @@ Template.reactiveTable.helpers({
         if (this.showNavigation === 'always') return true;
         if (this.showNavigation === 'never') return false;
         return getPageCount.call(this) > 1;
+    },
+    'getRowCount': function() {
+        var rowCount = getRowCount.call(this);
+        if (_.isObject(rowCount)) return rowCount.count;
+        else if (rowCount) return rowCount;
+        return '0';
```

Comment:
I think you're already handling the object somewhere else, so this function could be removed and it could just be `'getRowCount': getRowCount`